### DISCUSSION TITLE
GANomaly implementation difference from original repo

### DISCUSSION THREAD

**Discussion by mstrupp** [open-edge-platform/anomalib#1132] [General]
There is a slight difference between the implementation in anomalib and the [original implementation](https://github.com/samet-akcay/ganomaly). The first encoder convolution uses a padding of 1 in the original repo and a padding of 4 in anomalib.

[Original implementation line, 37](https://github.com/samet-akcay/ganomaly/blob/78da4ea9a99f5b02ab60dd651a18def929176d77/lib/networks.py#LL37C28-L37C28)
```python
nn.Conv2d(nc, ndf, 4, 2, 1, bias=False)
```

[Anomalib implementation, line 48](https://github.com/openvinotoolkit/anomalib/blob/4b45f0587425d1166b5c2b3e19735c89b2353718/src/anomalib/models/ganomaly/torch_model.py#LL48C20-L48C20)
```python
nn.Conv2d(num_input_channels, n_features, kernel_size=4, stride=2, padding=4, bias=False)
```

Isn't this relevant for the implementation?
